Dockerfile

```diff
@@ -9,8 +9,8 @@ WORKDIR /app
 RUN apt-get update && apt-get upgrade -y &&\
   apt-get install -y python3 python3-pip nano ffmpeg
 
+RUN python3 -m pip config set global.break-system-packages true
```

Comment:
Used to bypass the externally managed environment warning. While venv isolation makes sense on a bare metal environment, it does not seem necessary in a docker container.